server/controllers/groups.js

```diff
@@ -3,14 +3,15 @@
  */
 import firebase from 'firebase';
 import db from '../config/config';
+import Utils from '../utils/index';
 
 /**
  * @description create user group controller
  * @param {*} req
  * @param {*} res
  */
 export const createGroup = (req, res) => {
-  const groupname = req.body;
+  const groupname = req.body.groupname;
```

Comment:
this should work const {groupName} = req.body; Notice the camelCasing in groupName. This is the norm for javascript